### DISCUSSION TITLE
How to specify the titiler endpoint for cert?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [developmentseed/titiler#463] [Q&A]
We spun up a ti-tiler endpoint on AWS lambda, and it works fine with the long AWS generated URL.  

We then pointed the cname https://titiler.esipfed.org at the long AWS URL, but if we then hit that, we get a warning that it's not secure:
``` 
Websites prove their identity via certificates. Firefox does not trust this site because it uses a certificate that is not valid for titiler.esipfed.org. The certificate is only valid for *.execute-api.us-west-2.amazonaws.com.
```
How do we fix this?

**Comment by vincentsarago** [comment]
This is an AWS issue, you need to create a certificate for `titiler.esipfed.org` https://aws.amazon.com/certificate-manager/

**Comment by rsignell-usgs** [comment]
@vincentsarago we've struggled and failed to resolve this.   Could you put us in touch with someone who has ti-tiler running on AWS so we can get a bit more detailed instructions?  
@abarciauskas-bgse , @esip-lab

**Reply by vincentsarago** [reply]
I'm running TiTiler on aws with a Api gateway `Custom domain names` (https://titiler.xyz) I'm sorry but I again have to point you to the AWS documentation because I'm not 💯 sure of what's your issue and/or your AWS configuration.
